builtin/types/dposv3/dposv3.proto

```diff
@@ -57,9 +57,9 @@ message CandidateList {
 message Delegation {
     Address validator = 1;
     Address delegator = 2;
-    BigUInt amount = 3;
-    BigUInt update_amount = 4;
-    uint64 height = 5;
```

Comment:
We could keep the height, gives some metadata about when the Delegation happened, or we can add a `delegation_time` parameter so we can have better understanding of howl ong funds have been delegated for.